export_report.py

```diff
@@ -2,27 +2,55 @@
 
 import plotly.io as pio
 import pandas as pd
+from sqlalchemy import create_engine
 from docx import Document
 from docx.shared import Inches
 import os
 import re
+import subprocess
 import sys
 import datetime
 
 
-def generate_graphs():
+def generate_graphs(directory):
     graphs = {}
     # Walking through the folder to find all the result files
     # ignoring dirname since we don't need it in this implementation
-    for root, _, files in os.walk("."):
+    for root, _, files in os.walk(directory):
         for f in files:
```

Comment:
(nitpick) It's pretty common in Python to use `f` as a variable name to represent an actual file-like object vs. `filename` (or similar wording) when dealing with something else. Just something to take into consideration; no pressure to change.